dev/benchmarks/test_apps/stocks/lib/i18n/regenerate.md

```diff
@@ -1,29 +1,20 @@
 # Regenerating the i18n files
 
-The files in this directory are used to generate `stock_strings.dart`, which
-is used by the stocks application to look up localized message strings. The
-stocks app uses the [Dart `intl` package](https://github.com/dart-lang/intl).
+The files in this directory are used to generate `stock_strings[_locale].dart`
```

Comment:
This is a little confusing because "which is used" refers to one file. It might be simpler to just say that is generates `stock_strings.dart` which contains the StockStings class.